ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline using kfp_tekton python package
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/Pipelines/OpenShiftPipelines.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Library             ../../../../libs/DataSciencePipelinesKfpTekton.py
+Suite Setup         Data Science Pipelines Suite Setup
+Suite Teardown      RHOSi Teardown
```

Comment:
I think we would need a teardown to clean up the cluster from the pipeline resources which were created by the tests

Reply:
Done